arkane/mainTest.py

```diff
@@ -79,6 +80,29 @@ def test_arkane_examples(self):
             error_message += '{1} in {0}; '.format(example_name, example_type)
         self.assertFalse(self.failed, error_message)
 
+    def test_arkane_two_parameter_arrhenius_fit(self):
+        test_path = os.path.join(self.test_base_path, 'two_parameter_arrhenius_fit')
+        file_to_remove = ['output.py', 'chem.inp', 'supporting_information.csv']
+        for file in file_to_remove:
+            if os.path.exists(os.path.join(test_path, file)):
+                os.remove(os.path.join(test_path, file))
+        arkane = Arkane(input_file=os.path.join(test_path, 'input.py'), output_directory=test_path)
+        arkane.plot = False
+        arkane.save_rmg_libraries = False
+        arkane.execute()
+        with open(os.path.join(test_path, 'output.py'), 'r') as f:
+            output = f.readlines()
+        reverse_output = output[::-1]
+        for i, line in enumerate(reverse_output):
+            if 'kinetics fitted using' in line:
+                msg_output = line.rstrip()
+                n_output = reverse_output[i-5].split('=')[1].strip().replace(',', '')
+                break
+        msg_expected = '# kinetics fitted using the two-parameter Arrhenius equation k = A * exp(-Ea/RT)'
+        self.assertEqual(msg_output, msg_expected)
+        n_expected = '0'
+        self.assertEqual(n_output, n_expected)
+
     @classmethod
     def tearDownClass(cls):
```

Comment:
could you check whether running this test has leftover log files? If so, they should also be removed here under `tearDownClass`

Reply:
Now the test will delete leftover files. Thanks.